mcp/server/tools/src/services/AuthService.ts

```
@@ -0,0 +1,56 @@
+      if (!data) {
+        throw new Error('Invalid user login details')
+      }
+
+      const setCookieHeader = headers['set-cookie']?.[0]
+      if (setCookieHeader) {
+        authToken = setCookieHeader
+      } else {
+        throw new Error('No token cookie received')
+      }
+
+      this.authData = {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)